segment_spotify others.go

```diff
@@ -0,0 +1,8 @@
+// +build !darwin
```

Comment:
this file needs to be renamed to `segment_spotify_others.go `, notice the underscore.

Reply:
Sorry... I was half coding half watching football :sweat_smile: